data/allowlist.yaml

```diff
@@ -31,48 +62,1174 @@ bedrock_base_config: &bedrock_base_config
+
+    # Mozilla social accounts
+    - https://www.instagram.com/firefox/
+    - https://www.instagram.com/mozilla/
+    - https://www.instagram.com/unfcktheinternet/
+
+    # Specific YouTube links
+    - https://youtube.com/MozillaDeveloper
+    - https://www.youtube.com/t/terms
+    - https://www.youtube.com/user/firefoxchannel
+    - https://www.youtube.com/watch?v=6Ck1d6qo8a0
+    - https://www.youtube.com/watch?v=A_dZVbYJtsk
+    - https://www.youtube.com/watch?v=baPF-sOXP24
+    - https://www.youtube.com/watch?v=f_f5wNw-2c0
+    - https://www.youtube.com/watch?v=kw9LiqF-lNw
+    - https://www.youtube.com/watch?v=LuEX_Wih8g4
+    - https://www.youtube.com/watch?v=T2jykykN3yc
+    - https://www.youtube.com/watch?v=t3NKmmWfklU
+    - https://www.youtube.com/watch?v=ug6tA6fhhdQ
+    - https://www.youtube.com/watch?v=Un3u4wuGT8Q
+    - https://www.youtube.com/watch?v=zZzl649NcD0
+    - https://youtu.be/F-nFQryDB0s
+    - https://youtu.be/M9kBO5bIEyI?t=705
+
+    # Usenet links
+    - news://alt.fan.mozilla
+    - news://alt.netscape.sucks
+    - news://comp.infosystems.www.advocacy
+    - news://comp.lang.java.advocacy
+    - news://comp.os.linux.advocacy
+    - news://comp.os.ms-windows.advocacy
+    - news://comp.os.os2.advocacy
+    - news://comp.sys.amiga.advocacy
+    - news://comp.sys.be.advocacy
+    - news://comp.sys.mac.advocacy
+    - news://comp.unix.advocacy
+    - news://gnu.misc.discuss
+
+    # Malformatted or broken links which need fixing
+    - http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2013-1677)
+    - http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2013-1715)
+    - ./
+    - (https://www.mozilla.org/privacy/)
+    - "%(addons)s"
+    - "%(url)s"
+    - "%s"
+    - $https://developer.mozilla.org/docs/Glossary/HTML
+    - additivos
+    - applicationes
+    - chrome://global/skin/about.css
+    - congresso
+    - Contribute
+    - firefox-third-party.html
+    - "https: // www.google.com/privacy/lsf.html"
+    - incontros
+    - libro
+    - missing-var
+    - nb-NO
+    - Principles
+    - security@mozilla.org
+    - traciatores
+    - url
+    - url('mozorg.about.policy.transparency.index') + '#dfn-natl-security-request'
+    - varietate
+    - vr
+    - www.cybersensei.io
+    - www.securesoftware.nl
+    - " https://bugzilla.mozilla.org/buglist.cgi?f1=cf_tracking_fxios&list_id=12692445&o1=substring&resolution=---&resolution=FIXED&resolution=INVALID&resolution=WONTFIX&resolution=DUPLICATE&resolution=WORKSFORME&resolution=INCOMPLETE&resolution=SUPPORT&resolution=EXPIRED&resolution=MOVED&query_based_on=v1.2&query_format=advanced&bug_status=UNCONFIRMED&bug_status=NEW&bug_status=ASSIGNED&bug_status=REOPENED&bug_status=RESOLVED&bug_status=VERIFIED&bug_status=CLOSED&v1=1.2&product=Firefox%20for%20iOS&known_name=v1.2"
+    - " https://mullvad.net/help/why-wireguard/"
+    - https://android.googlesource.com/platform/frameworks/av/+/\nc87faed60483afb2466e03892bda80b72e5822c7%5E!/#F0
```

Comment:
This one will need the \n removing from the middle (`...av/+/\nc8...`) and the line moving in with the regular allowlist, once https://github.com/mozilla/foundation-security-advisories/pull/91/files is merged and the changed data has flowed up to bedrock production